noxfile.py

```diff
@@ -154,9 +155,11 @@ def coverage_release(session: nox.Session):
             "-e",
             f"antsibull_build_command={build_command!r}",
             env={
+                "TMPDIR": "/tmp",
```

Comment:
```suggestion
                "TMPDIR": os.environ.get("TMPDIR", "/tmp"),
```

is probably better in case the caller has a different TMPDIR set and /tmp isn't accessible. nox doesn't mess with the `TMPDIR` value in the global `os.environ` dict like it does with `session.run`.

Reply:
Ah, this line shouldn't be there anyway, I accidentally forgot to remove it. `TMPDIR` should be controlled by `**tmp_env` further below, which comes from the env variable.

Using general `TMPDIR` is IMO a bad idea, since that's a pretty common env var.

```suggestion
```